### PR TITLE
Removing `test_pfix()` from `workbooklib`

### PR DESCRIPTION
**Pull request by phildominguez-gsa**
Addresses https://github.com/GSA-TTS/FAC/issues/2672

In this PR:
- Removes `test_pfix()` from `workbooklib`, as it doesn't do anything
- Replacing some instances with `str` if a submission fails without it

Testing:
- `docker compose run web python manage.py historic_data_migrator --email <your email>`
  - If you get a login error, log into the app as normal and retry

## PR checklist: submitters

- [ ]   Link to an issue if possible. If there’s no issue, describe what your branch does. Even if there is an issue, a brief description in the PR is still useful.
- [ ]   List any special steps reviewers have to follow to test the PR. For example, adding a local environment variable, creating a local test file, etc.
- [ ]   For extra credit, submit a screen recording like [this one](https://github.com/GSA-TTS/FAC/pull/1821).
- [ ]   Make sure you’ve merged `main` into your branch shortly before creating the PR. (You should also be merging `main` into your branch regularly during development.)
- [ ]   Make sure you’ve accounted for any migrations. When you’re about to create the PR, bring up the application locally and then run `git status | grep migrations`. If there are any results, you probably need to add them to the branch for the PR. Your PR should have only **one** new migration file for each of the component apps, except in rare circumstances; you may need to delete some and re-run `python manage.py makemigrations` to reduce the number to one. (Also, unless in exceptional circumstances, your PR should not delete any migration files.)
- [ ]   Make sure that whatever feature you’re adding has tests that cover the feature. This includes test coverage to make sure that the previous workflow still works, if applicable.
- [ ]   Make sure the full-submission.cy.js [Cypress test](https://github.com/GSA-TTS/FAC/blob/main/docs/testing.md#end-to-end-testing) passes, if applicable.
- [ ]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step. If that’s not applicable for some reason, check this box.
- [ ]   Verify that no Git surgery was necessary, or, if it was necessary at any point, repeat the testing after it’s finished.
- [ ]   Once a PR is merged, keep an eye on it until it’s deployed to dev, and do enough testing on dev to verify that it deployed successfully, the feature works as expected, and the happy path for the broad feature area (such as submission) still works.

## PR checklist: reviewers

- [ ]   Pull the branch to your local environment and run `make docker-clean; make docker-first-run && docker compose up`; then run `docker compose exec web /bin/bash -c "python manage.py test"`
- [ ]   Manually test out the changes locally, or check this box to verify that it wasn’t applicable in this case.
- [ ]   Check that the PR has appropriate tests. Look out for changes in HTML/JS/JSON Schema logic that may need to be captured in Python tests even though the logic isn’t in Python.
- [ ]   Verify that no Git surgery is necessary at any point (such as during a merge party), or, if it was, repeat the testing after it’s finished.

The larger the PR, the stricter we should be about these points.

### CODE REVIEW

**Review by JeanMarie-TTS — COMMENTED** [review]
LGTM

**Review by JeanMarie-TTS — APPROVED** [review]

File: backend/census_historical_migration/workbooklib/secondary_auditors.py [hunk 1]
```diff
@@ -24,20 +23,6 @@
     FieldMap("secondary_auditor_name", "cpafirmname", "auditor_name", None, str),
     FieldMap("secondary_auditor_contact_phone", "cpaphone", "contact_phone", None, str),
     FieldMap("secondary_auditor_address_state", "cpastate", "address_state", None, str),
-    FieldMap(
```
Comment (sambodeme) [inline backend/census_historical_migration/workbooklib/secondary_auditors.py:27]: Why are these removed ?

**Review by sambodeme — CHANGES_REQUESTED** [review]

File: backend/census_historical_migration/workbooklib/secondary_auditors.py (hunk 1 above)
Comment (phildominguez-gsa) [reply backend/census_historical_migration/workbooklib/secondary_auditors.py]: Oops, fixed

File: backend/census_historical_migration/workbooklib/notes_to_sefa.py [hunk 2]
```diff
@@ -4,7 +4,6 @@
     set_single_cell_range,
     map_simple_columns,
     generate_dissemination_test_table,
-    test_pfix,
```
Comment (JeanMarie-TTS) [inline backend/census_historical_migration/workbooklib/notes_to_sefa.py:7]: Does this not have to be replaced with str, rather than merely removing test_pfix?

File: backend/census_historical_migration/workbooklib/secondary_auditors.py [hunk 3]
```diff
@@ -3,7 +3,6 @@
     set_uei,
     map_simple_columns,
     generate_dissemination_test_table,
-    test_pfix,
```
Comment (JeanMarie-TTS) [inline backend/census_historical_migration/workbooklib/secondary_auditors.py:6]: replace with str?

File: backend/census_historical_migration/workbooklib/notes_to_sefa.py (hunk 2 above)
Comment (phildominguez-gsa) [reply backend/census_historical_migration/workbooklib/notes_to_sefa.py:7]: Nope, `str` is a built-in Python [method](https://www.w3schools.com/python/ref_func_str.asp)

File: backend/census_historical_migration/workbooklib/secondary_auditors.py (hunk 3 above)
Comment (phildominguez-gsa) [reply backend/census_historical_migration/workbooklib/secondary_auditors.py:6]: Nope, see [other comment](https://github.com/GSA-TTS/FAC/pull/2790/files#r1393062897)

**Review by sambodeme — APPROVED** [review]
